CONTRIBUTING.md

```diff
@@ -0,0 +1,110 @@
+# How to Contribute
+
+*eksctl* is [Mozilla Public License Version 2.0 licenced](LICENSE.MPL-2.0) and accepts contributions via GitHub
```

Comment:
This is not quite true, `eksctl` is under [Apache 2.0 license](https://github.com/weaveworks/eksctl/blob/master/LICENSE). Only some vendored files are under MPL-2.0.

Reply:
I'll change that back to what it was...thanks.